subsys/bluetooth/common/dummy.c

```diff
@@ -28,3 +28,13 @@ BUILD_ASSERT(CONFIG_BT_HCI_TX_PRIO < CONFIG_BT_RX_PRIO);
  */
 BUILD_ASSERT(CONFIG_BT_CTLR_RX_PRIO < CONFIG_BT_HCI_TX_PRIO);
 #endif /* CONFIG_BT_CTLR */
+
+/* Immediate logging is not supported with the software-based Link Layer
+ * since it introduces ISR latency due to outputting log messages with
+ * interrupts disabled.
+ */
+#if !defined(CONFIG_TEST) && (defined(CONFIG_BT_LL_SW_SPLIT) || \
+			      defined(CONFIG_BT_LL_SW_LEGACY))
+BUILD_ASSERT_MSG(!IS_ENABLED(CONFIG_LOG_IMMEDIATE), "Immediate logging not \
+		 supported with the software Link Layer");
```

Comment:
Silly thing, these tabs are also printed, so maybe chop the string into 2 concatenated ones, i.e.
"Immediate logging not "
"supported with the software Link Layer"

Reply:
Changed that now